php/laravel/security/laravel-cookie-long-timeout.yaml

```diff
@@ -0,0 +1,35 @@
+rules:
+- id: laravel-cookie-long-timeout
+  patterns:
+  - pattern: |
+      'lifetime'
+  - pattern-inside: |
+      return [
+        ...,
+        'lifetime' => $TIME,
+        ...
+      ];
+  - pattern-not-inside: |
+      return [
+        ...,
+        'lifetime' => env("$VAR", $DEFAULT),
```

Comment:
we should compare the default to the lifetime as well and flag defaults that are wrong